.github/.codecov.yml

```diff
@@ -28,3 +28,9 @@ coverage:
         patch:
             default:
                 target: 85%
+
+odecov:
```

Comment:
Are you sure that the is no `c` missing here?

Reply:
thanks for spotting this, must be the reason why it did not work. I'll close this one and open a new one to test it.